---
layout: archive
title: "Publications"
permalink: /publications/
author_profile: true
---
# List of Publications
1. Liao, W., Ding, Y., **Shavit, Y.**, Huang, D., Ren, S., Guo, J., Feng, W. and Zhang, K. WT-MVSNet: Window-based Transformers for Multi-view Stereo. Accepted to NIPS2022 
2. **Shavit, Y**. and Keller, Y. Camera Pose Auto-Encoders for Improving Pose Regression. Accepted to ECCV22.
3. Shi, Y., Cai, J.X., **Shavit, Y**., Mu, T.J., Feng, W. and Zhang, K., 2022. ClusterGNN: Cluster-based Coarse-to-Fine Graph Neural Network for Efficient Feature Matching. In Proceedings of the IEEE/CVF Conference on Computer Vision and Pattern Recognition (pp. 12517-12526)
4. Fang, W., Zhang, K., **Shavit, Y**. and Feng, W., 2022. Adversarial Learning of Hard Positives for Place Recognition. arXiv preprint arXiv:2205.03871. ORAL at IJCNN22.
5. **Shavit, Y.**, Ferens, R., Keller, Y. (2021). Learning multi-scene absolute pose regression with transformers. IEEE International Conference on Computer Vision (ICCV). ORAL Presentation.
6. **Shavit, Y.**, Ferens, R., & Keller, Y. (2021). Paying Attention to Activation Maps in Camera Pose Regression. arXiv preprint arXiv:2103.11477. 
7. **Shavit, Y.**. and Klein, I., 2021. Boosting Inertial-Based Human Activity Recognition with Transformers. IEEE Access, 9, pp.53540-53547.
8. **Shavit, Y.** and Ferens, R. (2020) Do we really need scene-specific pose encoders? 2021 IEEE International Conference on Pattern Recognition (ICPR).
9. **Shavit, Y.**, & Ferens, R. (2019). Introduction to camera pose estimation with deep learning. arXiv preprint arXiv:1907.05272.
10. **Shavit, Y.**, Merelli, I., Milanesi, L., and Lio’, P. (2016). How computer science can help in understanding the 3D genome architecture. Brief Bioinformatics, 17:733-44.
11. **Shavit, Y.**, Walker, B., and Lio’, P. (2016). Hierarchical block matrices as efficient representations of chromosome topologies and their application for 3C data integration. Bioinformatics, 32:1121-9. 
12. **Shavit, Y.**, Yordanov, B., Dunn, S.J., Wintersteiger, C.M., Otani, T.,  Hamadi, Y., Livesey, F.J., Kugler, H. (2016) Automated Synthesis and Analysis of Switching Gene Regulatory Networks. Biosystems, 146:26-34.
13. Hamey, F., **Shavit, Y.**, Maciulyte, V., Town, C., Lio’, P., and Tosi, S. (2015). Automated Detection of Fluorescent Probes in Molecular Imaging. In: DI Serio C., Lio’, P., Nonis, A., and Tagliaferri, R., editors, Computational Intelligence Methods for Bioinformatics and Biostatistics, CIBB 2014, volume 8623 of Lecture Notes in Computer Science, pages 68–75. Springer International Publishing.
14. **Shavit, Y.**, Hamey, F. K., and Lio’, P. (2014). FisHiCal: an R package for iterative FISH-based calibration of Hi-C data. Bioinformatics, 30:3120–2.
15. **Shavit, Y.** and Lio’, P. (2014). Combining a wavelet change point and the Bayes factor for analysing chromosomal interaction data. Mol Biosyst, 10:1576–1585.
16. **Shavit, Y.** and Lio’, P. (2013). CytoHiC: a cytoscape plugin for visual comparison of Hi-C networks. Bioinformatics, 29:1206–1207.
17. Mashiach, E., Schneidman‐Duhovny, D., Peri, A., **Shavit, Y.**, Nussinov, R. and Wolfson, H.J., 2010. An integrated suite of fast docking algorithms. Proteins: Structure, Function, and Bioinformatics, 78(15), pp.3197-3204.
